spec/numberformat.html

```diff
@@ -328,6 +328,25 @@
         1. Return _m_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-unwrapnumberformat" aoid="UnwrapNumberFormat">
+      <h1>UnwrapNumberFormat(nf)</h1>
+      <p>
+        The UnwrapNumberFormat abstract operation gets the underlying NumberFormat operation
+        for various methods which implement ECMA-402 v1 semantics for supporting initializing
+        existing Intl objects.
+      </p>
+      <emu-alg>
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot,
+          1. If ? InstanceofOperator(_nf_, %NumberFormat%),
```

Comment:
I'm probably misreading this, but are you sure this shouldn't be "If _not_ ? InstanceofOperator(_nf_, %NumberFormat%),"?  It seems to me that if _nf_ is a NumberFormat, we should be returning it.  Only in the case where _nf_ isn't a NumberFormat, should we look at _nf_[`Intl.[[FallbackSymbol]]`].

And a similar change needs to be made, it seems to me, to the `Intl.NumberFormat` algorithm.

And similar changes would need to be made to the `DateTimeFormat` patch.

Again, probably misreading this, but I've been staring at it for a good ten minutes and can't parse it as anything other than doing nothing special if the object being acted upon is a non-`NumberFormat`.